lib/lotus.rb

```diff
@@ -9,4 +11,9 @@
 #
 # @see http://lotusrb.org
 module Lotus
+  # @since x.x.x
+
+  def self.env
+    @env ||= EnvironmentInqurier.new(Environment.new.environment)
```

Comment:
@nguyenngoc2505 This isn't thread safe. Can you please remove it the `@env ||=` part? Thank you :smile_cat: